Bluetooth: Mesh: Generic OnOff & Level Models demo

vikrant8052 opened the pull request:
This is a simple application demonstrating a Bluetooth mesh node in
which Root element has following models

- Generic OnOff Server
- Generic OnOff Client
- Generic Level Server
- Generic Level Client

And Secondary element has following models

- Generic OnOff Server
- Generic OnOff Client

Signed-off-by: vikrant 8051 <vikrant8051@gmail.com>

vikrant8052 committed:
Bluetooth Mesh OnFF & Level Models demo for nRF52840

This is a simple application demonstrating a Bluetooth mesh node in
which Root element has following models

- Generic OnOff Server
- Generic OnOff Client
- Generic Level Server
- Generic Level Client

And Secondary element has following models

- Generic OnOff Server
- Generic OnOff Client

Signed-off-by: vikrant more <vikrant8051@gmail.com>

vikrant8052 committed:
Bluetooth: Mesh: Generic OnOff & Level Models demo

This is a simple application demonstrating a Bluetooth mesh node in
which Root element has following models

 - Generic OnOff Server
 - Generic OnOff Client
 - Generic Level Server
 - Generic Level Client

And Secondary element has following models

 - Generic OnOff Server
 - Generic OnOff Client

Signed-off-by: vikrant 8051 vikrant8051@gmail.com

vikrant8051 commented:
@dbkinder Today I've tried my best to solve coding style related error . Hope now developers will review this PR. There is some error which I think irrelevant 

author email (vikrant8051 <vikrant8051@gmail.com>) does not follow the syntax: First Last <email>.

Is that mean I have to have create vikrant.more@gmail.com ??

vikrant8052 committed:
Bluetooth: Mesh: Generic OnOff & Level Models demo

This is a simple application demonstrating a Bluetooth mesh node in
which Root element has following models

 - Generic OnOff Server
 - Generic OnOff Client
 - Generic Level Server
 - Generic Level Client

And Secondary element has following models

 - Generic OnOff Server
 - Generic OnOff Client

Signed-off-by: vikrant more vikrant.more@gmail.com
